### DISCUSSION TITLE
Software vehicle reboot?

### DISCUSSION THREAD

**Discussion by cheralex** [bitcraze/discussions#122] [Q&A]
Dear CF team.
Is there any change to do a software reboot from internal application running on crazyflie? 
Sometimes I'd like to reboot a firmware remotely. Is it possible?

**Comment by tobbeanton** [comment]
It depends a bit on what you mean with a software reboot. Rebooting the full system nRF51+STM32F4 can be done but is not implemented. Doing a reboot/reset of the STM32F4 can be done with NVIC_SystemReset();

Rebooting remotely is possible using the cflib [PowerSwitch](https://github.com/bitcraze/crazyflie-lib-python/blob/master/cflib/utils/power_switch.py) utility class.
